src/JsonLDParser.ts

```diff
@@ -0,0 +1,39 @@
+import { Options } from './mergeHead';
+
+const parseSimple = function (jsonLd: Object, options: Options) {
+  const stringifiedJson = JSON.stringify(jsonLd, null, options.space);
+  const innerHTML = options.space === 0 ? stringifiedJson : `\n${stringifiedJson}\n`;
+  // FIXME: private api
+  const hid = `nuxt-jsonld-${this._uid}`;
+  return {
+    hid,
```

Comment:
Same `hid` will be generated for multiple script tags in this approach.
`hid` must be unique.